### PR TITLE
[doc] Introduce project OpenAOE

### PR DESCRIPTION
**Pull request by JiaYingLii**
## Motivation

OpenAOE mentioned.

### CODE REVIEW

File: docs/en/serving/restful_api.md [hunk 1]
```diff
@@ -148,6 +148,16 @@ You can also test restful-api through webui.
 lmdeploy serve gradio api_server_url --server-name ${gradio_ui_ip} --server-port ${gradio_ui_port}
 ```
 
+### webui through OpenAOE
+
+You can use [OpenAOE](https://github.com/InternLM/OpenAOE) for seamless integration with LMDeploy.
+
+```shell
+pip install -U openaoe 
+openaoe -f /path/to/your/config-template.yaml
```
Comment (AllentDan) [inline docs/en/serving/restful_api.md:157]: May provide a `config-template.yaml` or a link to get it.

Comment (AllentDan) [reply docs/en/serving/restful_api.md:157]: I see. [There](https://github.com/InternLM/OpenAOE/blob/main/docs/tech-report/config-template.md) it is.

File: docs/en/serving/restful_api.md [hunk 2]
```diff
@@ -148,6 +148,16 @@ You can also test restful-api through webui.
...
+```
+Please refer to the [guidance](https://github.com/InternLM/OpenAOE/blob/main/docs/tech-report/model_serving_by_lmdeploy/model_serving_by_lmdeploy.md) for more deploy information.
```
Comment (AllentDan) [inline docs/en/serving/restful_api.md:159]: more detailed?

**Review by AllentDan — APPROVED** [review]

**Comment by AllentDan** [comment]
May fix the lint error.

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**